src/main/java/core/basesyntax/Model/Cat.java

```diff
@@ -7,10 +7,16 @@
 
     @Override
     public String toString() {
-        return "Cat{" +
-                "age=" + age +
-                ", name='" + name + '\'' +
-                ", ownerName='" + ownerName + '\'' +
+        return "Cat{"
+                + "age="
+                + age
+                + ", name='"
+                + name
+                + '\''
+                + ", ownerName='"
+                + ownerName
+                + '\''
+                +
                 '}';
```

Comment:
It can be more compact
```suggestion
        return "Cat{"
                + "age=" + age
                + ", name='" + name + '\''
                + ", ownerName='" + ownerName + '\'' + '}';
```

Reply:
Completed